.changeset/foo-bar-ai.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-ai": minor
```

Comment:
Just curious, are we using "zero versioning" (e.g. `0.1.0`) until GA?

Reply:
Think that's the idea

Reply:
that was my thinking